### PR TITLE
Report lines starting with && in tidy

### PR DESCRIPTION
**Pull request by zbynekwinkler**
Partial implementation of the issue #10692 (the easy part).

---

This change is [Reviewable](https://reviewable.io/reviews/servo/servo/10751)

### CODE REVIEW

**Comment by zwn** [comment]
I see the CI is blocking on tidy returning 0. What should be the next step now? Should I go and clean up the instances? I am reluctant to do that since it will mess up blame (by replacing relevant info with irrelevant indentation changes).

**Comment by metajack** [comment]
If you change tidy you'll need to make tidy pass as well. You might make git blame slightly harder, but I think we all know how to follow history to get the real blame, so it's not a big concern.

**Comment by Ms2ger** [comment]
And it's not even 20 lines, so *shrug*

**Comment by zwn** [comment]
Anything else?

File: python/tidy/servo_tidy_tests/test_tidy.py [hunk 1]
```diff
@@ -61,6 +61,7 @@ def test_rust(self):
         self.assertEqual('extra space before :', errors.next()[2])
         self.assertEqual('use &[T] instead of &Vec<T>', errors.next()[2])
         self.assertEqual('use &str instead of &String', errors.next()[2])
+        self.assertEqual('operators should go at the end of the first line', errors.next()[2])
```
Comment (Wafflespeanut) [inline python/tidy/servo_tidy_tests/test_tidy.py]: Now that #10758 has landed, we should ensure that there are no more errors.

**Comment by zwn** [comment]
I've rebased it on current master and reordered the commits so that the check is enabled only after the files had been fixed.

File: python/tidy/servo_tidy_tests/test_tidy.py (hunk 1 above)
Comment (zwn) [inline python/tidy/servo_tidy_tests/test_tidy.py]: Done.

**Comment by Wafflespeanut** [comment]
Thanks for doing this! :)

@bors-servo r+

**Pull request merged**